src/clone.c

```diff
@@ -297,27 +390,53 @@ SYS_FUNC(clone3)
 	if (prefix_fun != tprint_value_changed_struct_begin)
 		tprint_struct_end();
 
+	if (show_namespace && tcp->u_rval >= 0) {
+		tcp->auxstr = get_namespace_auxstr(tcp->u_rval, arg.flags, tcp);
```

Comment:
The `SYS_FUNC(clone)` comment about `tcp->u_rval` also applies here.

Reply:
I think the most appropriate condition here would be `arg.flags & nsflags && !syserror(tcp)`.

Reply:
fixed locally.